app/lib/task/backend.dart

```diff
@@ -971,8 +972,9 @@ List<Version> _versionsToTrack(
     // Consider 5 latest major versions, if any:
     ...packageVersions
         // Ignore prereleases and retracted versions
-        .where((pv) => !pv.isRetracted && !pv.semanticVersion.isPreRelease)
-        .map((pv) => pv.semanticVersion)
+        .where((pv) => !(pv.retracted ?? false))
+        .map((pv) => Version.parse(pv.version))
```

Comment:
Why use `pv.version` over `pv.semanticVersion` ? I probably don't understand the difference.

Reply:
We are switching base classess here: `PackageVersion` did have `isRetracted`, the same field is `VersionInfo.retracted` and is nullable. Similarly: `PackageVersion` had `semanticVersion`, `VersionInfo` does only `version`.